ci/docker/runtime_functions.sh

```diff
@@ -43,11 +43,10 @@ scala_prepare() {
 
 check_cython() {
     set -ex
-    local python_ver=$1
-    local is_cython_used=$(python${python_ver} <<EOF
+    local is_cython_used=$(python3 <<EOF
```

Comment:
does this work with incomplete var?

Reply:
What do you mean? `check_cython` is only called at one place and I modified that place to not pass the `python_ver`.

Reply:
I thought it should be 
```suggestion
    local is_cython_used=$(python3) <<EOF
```

Reply:
I think the closing brace comes in line 52.